src/core/MeshTransmissionMaterial.tsx

```diff
@@ -119,6 +121,13 @@ class MeshTransmissionMaterialImpl extends THREE.MeshPhysicalMaterial {
         ...this.uniforms,
       }
 
+      // r153+ compat https://github.com/pmndrs/drei/issues/1482
+      shader.defines = { ...shader.defines }
+      Object.defineProperty(shader.defines, 'USE_ANISOTROPY', {
+        get() {},
+        set(_) {},
+      })
+
```

Comment:
This is the "proper" fix, but aliasing `anisotropy` to `anisotropicBlur` on the material also works. Removing this one.